jib-core/src/test/java/com/google/cloud/tools/jib/http/WithServerFailoverHttpClientTest.java

```diff
@@ -149,4 +150,43 @@ public void testProxyCredentialProperties()
       }
     }
   }
+
+  @Test
+  public void testVerbatimRedirectionUrls()
+      throws IOException, InterruptedException, GeneralSecurityException, URISyntaxException {
+    String url1 = "?id=301&_auth_=exp=1572285389~hmac=f0a387f0";
```

Comment:
how/where do we generate these values? Would be helpful for someone modifying these tests.

Reply:
For now this can be anything. Added the source anyway.